packages/eas-cli/src/commandUtils/easCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { Command } from '@oclif/command';
+
+import { AnalyticsEvent, flushAsync, initAsync, logEvent } from '../analytics';
```

Comment:
```suggestion
import { AnalyticsEvent, flushAsync as flushAnalyticsAsync, initAsync as initAnalyticsAsync, logEvent } from '../analytics';
```
or
```suggestion
import * as analytics from '../analytics';
```

Why? If you're familiar with this code and you see line 18 (https://github.com/expo/eas-cli/pull/437/files#diff-63e550a4862bf36242a53a0d7549240120c507876a295c6ebee9761c9f5fcbaeR18) you can assume this is a generic initializer for commands but if you follow the import you realize this is an analytics initializer.

Reply:
yeah this is a good point.